__tests__/src/core/coords.spec.js

```diff
@@ -0,0 +1,37 @@
+import Coords from '../../../src/core/coords';
+
+describe('Game', () => {
+    describe('Coords', () => {
+        test('should construct coords wtesth default value', () => {
+            const coords = new Coords();
+
+            expect(coords.y).toEqual(0);
```

Comment:
I think jest provides a `deepEqual` method

Reply:
.toEqual or .toMatchObject if you want match only a part of it.

Reply:
expect(coords).toEqual({x: 0, y: 0})